### DISCUSSION TITLE
[lyrics] export to LRC/TXT files

### DISCUSSION THREAD

**Discussion by ildar** [beetbox/beets#4596] [Ideas]
Many players can show lyrics from the files placed nearby, in `*.lrc` files.
How about to make lyrics plugin export to the `*.lrc` files besides RST as it is now?

**Comment by wisp3rwind** [comment]
Sounds like a useful feature; happy to take a PR that implements this!

**Comment by Dumuthy** [comment]
Did this ever get anywhere? I'm looking through the documentation and not finding anything related in the lyrics plugin. Perhaps a separate plugin was developed, or a community plugin?

**Comment by snejus** [comment]
We have an issue for this: https://github.com/beetbox/beets/issues/6396
